src/Wrapper.js

```diff
@@ -216,6 +221,7 @@ export default (Component) => {
   };
 
   WrappedComponent.defaultProps = {
+    innerRef: null,
```

Comment:
There was a small lint error here, where because innerRef was optional proptypes wanted a default prop. This fixes the lint error without any changes required elsewhere because you're already evaluating if the value of innerRef is truthy and null evaluates as false.